### PR TITLE
OIDC FAPI Conformance GitHub action build onboarding

### PR DESCRIPTION
**Pull request by anjuchamantha**
With this PR, OIDC FAPI Conformance suite is onboarded as a GitHub action build to this product-is repo. 

Issue: https://github.com/wso2/product-is/issues/16630

### CODE REVIEW

File: oidc-fapi-conformance-tests/test_runner_fapi.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# echo
+# python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=private_key_jwt][fapi_profile=plain_fapi][fapi_response_mode=jarm][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_pvtkeyjwt.json 2>&1 | tee fapi-testplan-pvtkeyjwt-jarm-log.txt
+# echo
+
+# echo
+# echo "FAPI Basic test plan - mtls, plain_fapi, jarm, by_value"
+# echo "-----------------------------"
+# echo
+# python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=mtls][fapi_profile=plain_fapi][fapi_response_mode=jarm][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_mtls.json 2>&1 | tee fapi-testplan-mtls-jarm-log.txt
+# echo
```
Comment (Thumimku) [inline oidc-fapi-conformance-tests/test_runner_fapi.sh:53]: EOL - check other places

File: oidc-fapi-conformance-tests/test_runner_fapi.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+echo
+
+echo
+echo "FAPI Basic test plan - mtls, plain_fapi, plain_response, by_value"
+echo "-----------------------------"
+echo
+python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=mtls][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_mtls.json 2>&1 | tee fapi-testplan-mtls-log.txt
+echo
+
+# echo
```
Comment (Thumimku) [inline oidc-fapi-conformance-tests/test_runner_fapi.sh:27]: Why this is commented?

**Comment by anjuchamantha** [comment]
fapi-action

File: oidc-fapi-conformance-tests/test_runner_fapi.sh (hunk 2 above)
Comment (anjuchamantha) [reply oidc-fapi-conformance-tests/test_runner_fapi.sh:27]: Currently the PAR and JARM FAPI profiles are not passing, and hence I have added the lines and commented so we just need to uncomment those 2 lines when those profiles are ready.

JARM issue: https://github.com/wso2/product-is/issues/18062

File: oidc-fapi-conformance-tests/test_runner_fapi.sh (hunk 1 above)
Comment (Thumimku) [inline oidc-fapi-conformance-tests/test_runner_fapi.sh:53]:
```suggestion
# echo

```

Comment (anjuchamantha) [reply oidc-fapi-conformance-tests/test_runner_fapi.sh]: Fixed with [5655e1a](https://github.com/wso2/product-is/pull/18059/commits/5655e1a6d406c0596f517b3e262483b0a3032be7)

Comment (anjuchamantha) [reply oidc-fapi-conformance-tests/test_runner_fapi.sh]: Fixed with [5655e1a](https://github.com/wso2/product-is/pull/18059/commits/5655e1a6d406c0596f517b3e262483b0a3032be7)

**Review by Thumimku — APPROVED** [review]
Approving this based on the initial review, please do the improvements mentioned in the code review.

**Pull request merged**